torch/utils/tensorboard/summary.py

```diff
@@ -459,3 +461,92 @@ def compute_curve(labels, predictions, num_thresholds=None, weights=None):
     precision = tp / np.maximum(_MINIMUM_COUNT, tp + fp)
     recall = tp / np.maximum(_MINIMUM_COUNT, tp + fn)
     return np.stack((tp, fp, tn, fn, precision, recall))
+
+
+def _get_tensor_summary(name, display_name, description, tensor, content_type, json_config):
+    """Creates a tensor summary with summary metadata.
+
+    Args:
+      name: Uniquely identifiable name of the summary op. Could be replaced by
+        combination of name and type to make it unique even outside of this
+        summary.
+      display_name: Will be used as the display name in TensorBoard.
+        Defaults to `tag`.
+      description: A longform readable description of the summary data. Markdown
+        is supported.
+      tensor: Tensor to display in summary.
+      content_type: Type of content inside the Tensor.
+      json_config: A string, JSON-serialized dictionary of ThreeJS classes
+        configuration.
+
+    Returns:
+      Tensor summary with metadata.
+    """
+    import torch
+    tensor = torch.as_tensor(tensor)
+
+    tensor_metadata = SummaryMetadata(plugin_data=SummaryMetadata.PluginData(
```

Comment:
I didn't run the code yet. So I just compared the code your code with https://github.com/tensorflow/tensorboard/blob/38749e71942a6a5595a282597ba80e92d6721f9c/tensorboard/plugins/mesh/metadata.py#L68
It looks like many parameters are different. Because you didn't push the proto you used, I can not further check.

Reply:
What do you mean by proto and where should I push it?

I updated the function based on the code you linked.

Reply:
Hi, I have fixed the critical part. The following is the result of your sample code.
![image](https://user-images.githubusercontent.com/2005323/57973464-a71cec00-79db-11e9-9e7c-319838b566c6.png)